src/sentry/conf/types/kafka_definition.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from collections.abc import Callable, Mapping, Sequence
+from enum import Enum
+from typing import Any, Required, TypedDict
+
+import click
+
+
+class Topic(Enum):
```

Comment:
It would be useful to add a docstring here explaining that these are the logical topics. This is because here is where people add topics, this is being used by many.

Reply:
Added docstring. There is also a test now that asserts that these same values are present in sentry-kafka-schemas